crates/pcb/src/release.rs

```diff
@@ -345,11 +346,12 @@ fn git_version_and_hash(path: &Path) -> Result<(String, String)> {
         let tags = String::from_utf8(tag_out.stdout)?;
         let tags: Vec<&str> = tags.lines().collect();
 
-        // Use the first tag if any exist
-        if let Some(tag) = tags.first() {
-            if !tag.is_empty() {
-                let version = tag.to_string();
-                info!("Git version (tag): {version}");
+        // Look for board-specific tag in format "board_name/version"
+        let tag_prefix = format!("{board_name}/");
```

Comment:
what do you think about making the board name comparison case-insensitive?

Reply:
done